src/app/core/site-header/site-header.component.html

```diff
@@ -2,12 +2,21 @@
   <nav class="navbar navbar-expand-lg">
     <div class="container">
       <div class="d-flex flex-row">
-        <img class="navbar-brand herb" [src]="'herb.png' | path: 'imagesPath'" alt="herb" />
+        <a href="/">
+          <span class="visually-hidden">{{ "MAIN_PAGE" | translate }}</span>
+          <img class="navbar-brand herb" [src]="'herb.png' | path: 'imagesPath'" alt="herb" />
+        </a>
         <a href="https://www.gov.pl/web/premier" class="navbar-brand">
-          <img class="logo" [src]="'kprm.svg' | path: 'imagesPath'" alt="kancelaria prezesa rady ministrów" />
+          <span class="visually-hidden">{{ "CHANCELLERY_OF_PRIME_MINISTER" | translate }}</span>
+          <img
+            class="logo"
+            [src]="'kprm.svg' | path: 'imagesPath'"
+            [alt]="'CHANCELLERY_OF_PRIME_MINISTER' | translate"
+          />
         </a>
-        <a href="https://pomagamukrainie.gov.pl/" class="navbar-brand">
-          <img class="logo" [src]="'pomagamyua.svg' | path: 'imagesPath'" alt="pomagamy ukrainie" />
+        <a href="/" class="navbar-brand">
```

Comment:
Shouldn't this link to pomagamukrainie.gov.pl?

Reply:
That’s exactly this site; why would we need an absolute URL?

Reply:
Now we have two links in the header pointing at pomagamukrainie.gov.pl and I'm just wondering if that's fine

Reply:
we will discuss later, it seems that the first link is gov.pl

Reply:
Yes, I voiced exactly this concern in the JIRA that it seems weird to have these two links pointing to the same URL. I can change the first one to `gov.pl`.

Reply:
I swapped it to `gov.pl`; if the final decision is different, we can update it later.